Assignment 2.Rmd

```diff
@@ -87,14 +93,70 @@ plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender, edge.widt
 
 In Part II your task is to [look up](http://igraph.org/r/) in the igraph documentation and create a graph that sizes the student vertices in terms of the number of disciplinary actions they have recieved, and the teachers in terms of the number of disciplinary actions they have given out.
 
+``` {r}
+library(tidyr)
+library(dplyr)
+
+STU.SUM <- EDGE%>%group_by(to)%>%summarise(sum(count)) 
+TE.SUM <- EDGE%>%group_by(from)%>%summarise(sum(count)) 
+names(STU.SUM) <- c("id","count")
+names(TE.SUM) <- c("id","count")
+TOTAL <- bind_rows(STU.SUM,TE.SUM)
+VERTEX2 <- full_join(VERTEX,TOTAL,by="id")
 
```

Comment:
Make sure your variables are of the same type when you bind (`factor` to `factor`, not `factor` to `character`).